astroplan/tests/test_telescope.py

```diff
@@ -0,0 +1,200 @@
+                       399.57293121, 409.3114413, 419.5961146, 429.14136695,
+                       439.52687038, 449.9795939, 459.69289647, 469.60785929,
+                       479.85892255, 489.5638226, 499.59835962, 509.99161922,
+                       519.5607864, 529.30446727, 539.85285015, 549.59976275,
+                       559.51472558, 569.94676599, 579.68075166, 589.59571448,
+                       599.9631202, 609.56008031, 619.65269622, 630.09581687,
+                       639.67467926, 649.50561697, 659.84070534, 669.7685951,
+                       679.50258077, 689.9637068, 699.78494931, 709.53555533,
+                       719.83463294, 729.72858948, 739.88431656, 749.9673296,
+                       759.66253445, 769.59042422, 780.37854306, 789.59647942,
+                       799.60677842, 810.07759966, 819.65646205, 829.52341053,
+                       839.82248813, 849.68943661, 859.77244965, 870.07152726,
+                       879.71760973, 889.81096433, 899.94245339, 909.73137855,
+                       919.69435573, 930.06545485, 939.64431724, 949.72733028,
+                       959.95862293, 969.6772918, 979.76030484, 990.05938245,
+                       1100.]) * u.nm
+        response = np.array([0., 70.30747425, 73.11442327, 75.26541144,
+                             76.58538173, 74.70200949, 77.10351031,
+                             81.79762371, 84.528972, 87.18136276,
+                             89.0405892, 91.16038906, 92.49142617,
+                             93.66048522, 94.87582372, 95.64295585,
+                             96.56602958, 97.20551595, 98.01709918,
+                             98.32588679, 98.26189462, 98.79719419,
+                             99.10133838, 99.09379281, 99.35788748,
+                             99.30100555, 99.14661175, 98.81209184,
+                             98.74843825, 98.30855134, 98.04495971,
+                             97.98459522, 97.24513015, 96.85779131,
+                             96.40002722, 96.03435769, 95.87183789,
+                             95.09275863, 94.89671913, 94.49854563,
+                             94.12126754, 93.52139537, 92.57268607,
+                             91.77633908, 90.30410094, 88.68846298,
+                             86.74856762, 84.85909033, 82.85400237,
+                             80.76562301, 78.18504085, 75.90628116,
+                             72.63150731, 68.93418199, 65.34249453,
+                             61.79608045, 58.13799205, 54.49429826,
+                             49.87975185, 45.18326838, 41.57397462,
+                             36.82027063, 32.80603172, 28.7917928,
+                             25.09446748, 21.39714216, 18.4392819,
+                             14.89286782, 0.]) * u.Unit('') / 100
+        default_ccd_response = (wl, response)
+        TELE = Telescope(self.diameter, self.bandpass, self.gain,
```

Comment:
Small style point: here and elsewhere, please use lowercase `tele` as the instance of the `Telescope` object, we generally reserve all caps names for global variables.